cypress/fixtures/login.json

```diff
@@ -1,3 +1,5 @@
 {
+  "user_name": "admin",
+  "password": "X4YRyeg7ES",
```

Comment:
Please don't include credentials in these scripts, unless they are dummy credentials from a mocked API or DB.

Reply:
I modified the script's credentials.